v2/components/schemas/QuotaGroup.yaml

```
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: object
+description: Defines a resource quota group for one or more snaps.
+properties:
+  group-name:
+    type: string
+    description: The name of the quota group.
+    example: 'logmem'
```

though both are equivalent AFAIU, we should pick the format that appears most often in the docs and use it consistently
```suggestion
    example: logmem
```